router/worker.go

```diff
@@ -696,6 +639,11 @@ func (w *worker) processDestinationJobs() {
 		status.ErrorResponse = routerutils.EnhanceJSON(routerutils.EmptyPayload, "response", routerJobResponse.respBody)
 		status.ErrorCode = strconv.Itoa(respStatusCode)
 
+		if isSuccessStatus(respStatusCode) {
+			successCount++
+		} else {
+			errorCount++ // TODO: shall we treat aborted as errors?
```

Comment:
I think if we are flushing the jobs out of the system we shouldn't be much concerned about limiting them or circuit breaking